### DISCUSSION TITLE
Pausing Time Tracker During Session

### DISCUSSION THREAD

**Discussion by Perry8889** [immersivecognition/unity-experiment-framework#116] [Q&A] [answered]
Hello!

I've got a scenario in my experiment where the participant may want to look at some help instructions to refresh their memory of what their task consists of. In this situation, I would like to pause the timer so their task completion times aren't artificially inflated by re-reading the instructions. 

Is this possible? I'm having some trouble figuring out how to implement that programatically.

Thank you!

**Comment by jackbrookes** [comment, marked as answer]
Hi! UXF doesn't have a way to do this directly, but there are a few ways you could solve your issue:

1. You could set `Time.timeScale = 0f;` in a script when you launch the instructions. In Unity this effectively "pauses" the game, and the times that are captured in UXF e.g. trial `start_time` will be paused.
2. You could just record the amount of time they are reading the instructions, and subtract this in your data analysis stage. This may be preferable because you may want to know if there is a relationship between amount of time reading instructions and performance,
3. You could make it so that the instructions are only available between trials, not within a trial. So then you only count the time they spent in the trial.

Hope that helps!

**Reply by Perry8889** [reply]
Thank you! Just tested out your first suggestion with Time.timeScale and this worked perfectly for my needed solution.
